$OpenBSD: patch-i3bar_src_child_c,v 1.6 2014/01/19 10:25:30 dcoppa Exp $

commit 612d25c12cd1bd9ff4167e8530fe9a2c54a619e4
Author: Tony Crisci <tony@dubstepdish.com>
Date:   Tue Jan 7 13:32:21 2014 -0500

i3bar Bugfix: don't show "EOF" status line error

When the `status_command` sends EOF, it is terminated. Terminating this
process prints an error message to the status line (hence, a race
condition). This error message is always more useful than the former
"EOF" status line error because it shows the exit code.

--- i3bar/src/child.c.orig	Sat Jan 18 22:48:02 2014
+++ i3bar/src/child.c	Sat Jan 18 22:48:25 2014
@@ -294,8 +294,6 @@ static unsigned char *get_buffer(ev_io *watcher, int *
             /* end of file, kill the watcher */
             ELOG("stdin: received EOF\n");
             cleanup();
-            set_statusline_error("Received EOF from statusline process");
-            draw_bars(false);
             *ret_buffer_len = -1;
             return NULL;
         }
